common/src/main/java/io/quarkus/ts/openshift/common/util/AwaitUtil.java

```diff
@@ -64,7 +64,7 @@ public void awaitReadiness(List<HasMetadata> resources) {
                     System.out.println(ansi().a("waiting for ").a(readableKind(it.getKind())).a(" ")
                             .fgYellow().a(it.getMetadata().getName()).reset().a(" to become ready"));
                     await().pollInterval(1, TimeUnit.SECONDS)
-                           .atMost(5, TimeUnit.MINUTES)
+                           .atMost(10, TimeUnit.MINUTES)
```

Comment:
I have to increase this timeout as it tends to spend more time downloading artifacts from artifactory.

Reply:
I'd suggest doing something like this: https://github.com/thorntail/openshift-test/commit/0ca7ba4d60ef3742b5a18d6a606ea099db8df589

Reply:
It makes totally sense. Will add it. Thanks for sharing!

Reply:
Added.